src/picterra/client.py

```diff
@@ -76,16 +76,23 @@ class ResultsPage(Generic[T]):
     """
     Interface for a paginated response from the API
 
-    Typically the endpoint returning list of objects return them splitted
-    in pages (page 1, page 2, etc..) of a fixed dimension (eg 20). Thus
-    each `list_XX` function returns a ResultsPage (by default the first one);
-    once you have a ResultsPage for a given list of objects, you can:
-    * check its length with `len()` (eg `len(page)`)
-    * access a single element with the index operator `[]` (eg `page[5]`)
-    * turn it into a list of dictionaries with  `list()` (eg `list(page)`)
-    * get the next page with `.next()` (eg `page.next()`); this could return
-    None if the list is finished
-    You can also get a specific page passing the page number to the `list_XX` function
+    Typically the Picterra API HTTP endpoints that list objects return
+    them splitted in 1-based indexed pages of a fixed dimension
```

Comment:
Sorry to intrude, grammar nitpick from me too: `s/splitted in/split into/`

Reply:
done